tests/test_buku.py

```diff
@@ -87,9 +87,9 @@ def test_get_PoolManager(m_myproxy):
 @pytest.mark.parametrize(
     "keywords, exp_res",
     [
-        ("", DELIM),
-        (",", DELIM),
-        ("tag1, tag2", ",t a g 1,t a g 2,"),
+        ([""], DELIM),
+        ([","], DELIM),
+        (["tag1, tag2"], ",tag1,tag2,"),
```

Comment:
The docs specify it must be a list… and what sort of logic is "put spaces after every character in tag names" anyway? :sweat_smile: